docs/documentation/pack-docs.md

```diff
@@ -87,7 +87,7 @@ An example of including a YouTube video can be seen at the [Zoom Feed README](ht
 In order to classify packs and make them easier to find, there are a number of pack metadata elements that you can use:
 - ***Use-case*** -- a name of a situation in which packs can be used - usually the pack name. The optional approved values for use-cases are:
 
-- ***Tags*** -- words from the same semantic field of the use-case. These make it easier to find packs using filters or the search bar, and are visible on the screen to help understand what the pack is and its benefit to users. The optional approved values for tags are:
+- ***Tags*** -- words from the same semantic field of the use-case. These make it easier to find packs using filters or the search bar, and are visible on the screen to help understand what the pack is and its benefit to users. To add a unique tag for a marketplace, add the marketplace name as a prefix for the tag name, as in the following example: ['marketplace_name:tag_name']. The tags under the 'Common tags' are valid for all marketplaces, and do not require a specific prefix. The optional approved values for tags for each marketplace are:
```

Comment:
I am unsure as to what the **optional approved values for tags for each marketplace are**.   If it is meant to be keywords, categories, etc., then they need to be indented.  If it is something else, then the list is missing.  Basically, this is not clear.

Reply:
The list will be attached under this text, this is the list: 
<img width="839" alt="image" src="https://user-images.githubusercontent.com/93524335/205001436-77dd7e59-8a49-49c6-9168-ddac9fff039d.png">
<img width="800" alt="image" src="https://user-images.githubusercontent.com/93524335/205001481-8ea0e9eb-d3d2-48db-bc89-e4a0db4d0529.png">

Reply:
@MLainer1 Great.  That makes more sense.